src/Accelerators/NNPA/Compiler/NNPACompilerOptions.cpp

```diff
@@ -55,6 +55,13 @@ llvm::cl::opt<bool> nnpaEnableCompilerStickUnstick(
                    "stick/unstick code. Default is false."),
     llvm::cl::init(false), llvm::cl::cat(OnnxMlirOptions));
 
+llvm::cl::opt<bool> nnpaEnableScalarBcastBinary(
+    "nnpa-enable-scalar-bcast-binary",
+    llvm::cl::desc("Enable the lowering to NNPA the broadcasting binary ops "
```

Comment:
I would suggest a clearer wording:

```
Enable the lowering to NNPA of binary operations with broadcasting of a scalar operand. Currently only enable ONNXDiv. Default is false.
```

Reply:
> I assume it is to cover a regression. 

Yes. This was a cherry pick of https://github.com/onnx/onnx-mlir/pull/2778 into our upcoming 0.4.2.0 release. It resolves the performance regression for roberta-sequence-classification-9 from https://github.com/onnx/onnx-mlir/issues/2769.

Reply:
> Are there examples where this optimization is worth it? And others where it is not? It feel that it could use a performance model.

I was effective to roberta-base, but later a rule to propagate Div to the inputs of MatMul can remove Div.
I don't have a model to which this feature is effective now, but in general, we want to offload a broadcasting div to NNPA only if it is surrounded by NNPA ops. 

I reworded the explanation in PR #2780.